build.yaml

```diff
@@ -55,11 +55,18 @@ build-env:
     cd /
 
     # build lxc
+    apk add meson ninja docbook2x docbook2x-doc curl
     git clone --depth 1 ${{LXC_CLONE_URL}} -b ${{LXC_BRANCH}}
     cd lxc
-    ./autogen.sh
-    ./configure --enable-static-binaries --prefix=/usr
-    make -j$(grep -c processor /proc/cpuinfo) install
+    mkdir /etc/default
+    # patches
+    ## static lib name
+    curl -L https://github.com/lxc/lxc/commit/64eb31d02d4933ad414239130df3dff45a0f6f91.diff | git apply
+    ## date check
+    sed -i '311s/$/.strip()/' meson.build
```

Comment:
does this have an upstream issue or pr associated with it? Just reading, I have no context on why this is here.
Is this the change you were actually intending to make?

```
$ git checkout upstream/stable-5.0 
HEAD is now at 116c24ade Merge pull request #4215 from cyphar/lxc5.x-opensuse-build
$ sed -i '311s/$/.strip()/' meson.build
$ git diff
diff --git a/meson.build b/meson.build
index 93572975e..59c386bec 100644
--- a/meson.build
+++ b/meson.build
@@ -308,7 +308,7 @@ if not want_sd_bus.disabled()
             oss_fuzz_dependencies += libsystemd
         endif
     endif
-
+.strip()
     srcconf.set10('HAVE_LIBSYSTEMD', has_sd_bus)
 else
     has_sd_bus = false
```

Reply:
perhaps its related to https://github.com/lxc/lxc/pull/4224 . thanks for filing an upstream issue and PR, please put a reference in here if that is what you're fixing.